maintainers/scripts/luarocks-packages.csv

```diff
@@ -78,10 +79,11 @@ mpack,,,,,
 moonscript,,,,,arobyn
 nvim-client,,,,,
 penlight,,,,,
+plenary.nvim,,,,lua5_1,
 rapidjson,,,,,
 readline,,,,,
 say,,,,,
-std__debug,std._debug,,,,
+std-_debug,std._debug,,,,
```

Comment:
normalization is not fully working.

Reply:
what do you mean ? original package name is `std._debug` it gets renamed to `std-_debug`.